contracts/chainlink-keepers/AutomaticCommitVolKeeper.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.0;
+
+import "./KeeperCompatibleInterface.sol";
+import "../series/IVolatilityOracle.sol";
+import "../configuration/IAddressesProvider.sol";
+import "../series/IPriceOracle.sol";
+
+contract AutomaticClaimVolKeeper is KeeperCompatibleInterface {
```

Comment:
The name of the contract doesn't reflect what it does (it doesn't claim anything). Should it be just `VolatilityOracleKeeper`?